### PR TITLE
Add GitHubSearch by @Karambirov

### PR DESCRIPTION
**Pull request by Karambirov**
1. [x] Project URL: https://github.com/Karambirov/GitHubSearch
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -17624,6 +17624,26 @@
...
       "stars": 203
+    },
+    {
+      "title": "GitHubSearch",
+      "category-ids": [
+        "github", 
+        "realm", 
+        "developer"
+      ],
+      "description": "GitHub client written using MVVM architecture with Router (without Rx and Interface Builder).",
```
Comment (vpeschenkov) [inline contents.json]: @Karambirov Hi! Remove (.) the period from the end of description, please.

File: contents.json [hunk 2]
```diff
@@ -17624,6 +17624,26 @@
       "date_added": "Apr 8 2019",
       "suggested_by": "@vpeschenkov",
       "stars": 203
+    },
+    {
+      "title": "GitHubSearch",
+      "category-ids": [
+        "github", 
+        "realm",
```
Comment (dkhamsing) [inline contents.json]: Can you remove realm and developer, let’s put this in a single category. Thx

Comment (Karambirov) [reply contents.json]: Done

File: contents.json (hunk 1 above)
Comment (Karambirov) [reply contents.json]: Done

**Comment by vpeschenkov** [comment]
@Karambirov thank you!

**Comment by dkhamsing** [comment]
Thank you !

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**